### PR TITLE
Switch "Local path" field to copy button instead of open new button

### PR DESCRIPTION
**Pull request by danielbachhuber**
## Proposed Changes

Switches the "Local path" field to use a copy button instead of open new button.

**Before:**

CleanShot 2024-09-10 at 03 48 40@2x

**After:**

CleanShot 2024-09-10 at 03 47 45@2x

## Testing Instructions

1. Open the "Settings" tab.
2. Verify clicking the copy icon adds the site's path to the clipboard.

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@matt-west It seems odd that "copy" is the default behavior for the other fields:

CleanShot 2024-09-03 at 04 59 24@2x

What do you think about making "copy" the default behavior for this field too, and then using an "open in new window" icon for opening the file path?

**Comment by matt-west** [comment]
@danielbachhuber Yep, that works. Thanks for the suggestion!

**Comment by wojtekn** [comment]
@danielbachhuber  @matt-west Studio already provides a way to open a directory in the Overview tab, 'Open in...' -> 'Finder' option.

What if we make it consistent in the Settings tab and use the copy action for all listed items, ditching the open action there?

**Comment by danielbachhuber** [comment]
> What if we make it consistent in the Settings tab and use the copy action for all listed items, ditching the open action there?

@wojtekn I'm amenable to this. I tried using the copy button and then open new, but it looked funky when you clicked the button and "Copied!" appeared.

**Comment by wojtekn** [comment]
Thanks @danielbachhuber. Yes, I agree that two buttons together could be confusing there.

**Comment by matt-west** [comment]
Do you think users will need to copy the local path more than they want to open the folder in Finder?

I agree that we can simplify to one action here. Copying makes sense for the other items in settings as it's things like passwords, URLs, and username. I’m not yet convinced that it needs to be the primary action for local path though. Can you share some example of when users will need to copy the path as text.

**Comment by wojtekn** [comment]
> Do you think users will need to copy the local path more than they want to open the folder in Finder?

They can open the folder in the Overview tab.

> I agree that we can simplify to one action here. Copying makes sense for the other items in settings as it's things like passwords, URLs, and username. I’m not yet convinced that it needs to be the primary action for local path though. Can you share some example of when users will need to copy the path as text.

Now, users need to open the Terminal and run `pwd` command to copy the path. An example use case is when users want to run any command line that requires the absolute path, e.g., copy the site to another directory, backup the directory, or scan it using a tool like lint.

File: src/components/content-tab-settings.tsx
Comment (wojtekn) [inline src/components/content-tab-settings.tsx:1]: Let's remove no longer used imports - `Icon`, `file`, `getIpcApi`, and `Button`.

**Review by wojtekn — APPROVED** [review]

**Comment by danielbachhuber** [comment]
> > Can you share some example of when users will need to copy the path as text.
> 
> Now, users need to open the Terminal and run `pwd` command to copy the path. An example use case is when users want to run any command line that requires the absolute path, e.g., copy the site to another directory, backup the directory, or scan it using a tool like lint.

This is what I ran into. I'm not sure of a use case to open the site's directory in the Finder, to be honest.

**Comment by wojtekn** [comment]
>  I'm not sure of a use case to open the site's directory in the Finder, to be honest.

I use that occasionally to edit or delete files or review the site content quickly.

I think that with your change, the UI is clearer as:
- all actions that open the browser or another app are kept in the Overview tab
- all actions in Settings allow to copy the text

**Comment by matt-west** [comment]
Thanks for the additional info.

Happy to drop the folder icon and go with the copy action instead. 👍

I’ve updated the designs in Figma to reflect the change.

**Pull request merged**